.gitignore

```diff
@@ -1,12 +1,10 @@
 # PHPUnit
 /phpunit.xml
 coverage/
+/test/Functional/Fixtures/cache
+/test/Functional/Fixtures/logs
 
 # composer
 /vendor/
 /composer.phar
```

Comment:
I propose to remove this one as well, for the same reasons (and because you should install composer globally).